poleno/utils/tests/test_templatetags.py

```diff
@@ -199,18 +201,32 @@ def language_view(request):
             u'request': request,
         })))
 
-    urls = tuple(patterns(u'',
+    urlpatterns = tuple(patterns(u'',
         url(r'^$', active_view, name=u'index'),
         url(r'^first/', active_view, name=u'first'),
         url(r'^second/', include(namespace=u'second', arg=patterns(u'',
             url(r'^$', active_view, name=u'index'),
             url(r'^first/', active_view, name=u'first'),
         ))),
     ))
-    urls += tuple(i18n_patterns(u'',
+    urlpatterns += tuple(i18n_patterns(u'',
         url(r'^language/', language_view, name=u'language'),
     ))
 
+    @staticmethod
+    def page_not_found(request):
+        template = Template(
+            u'{% load change_lang from poleno.utils %}'
+            u'{% change_lang lang.0 %}'
```

Comment:
@martinmacko47 Vysvetlil by si mi prosim ako funguje to cislovanie `.0` a pod. Aj s roznymi cislami, aj bez cisla to fungovalo rovnako. Vzdy sa redirectol na `en/language/`

Reply:
@viliambalaz Neviem. Odkial mas tu premennu `lang`? Ako sa dostala do kontextu? Ak si ju skopiroval z `main/snippets/header/language.html`, tak tam `lang` je z `{% for lang in LANGUAGES %}`. Teda je to dvojica tvaru `(u'en', u'English')` a teda `lang.0` je to `en`.

Tu v tvojom template to skor vyzera, ze `lang` nemas ani definovany. Takze `lang.0` bude proste `None` alebo prazdny string. Skor chces v tom teplate pouzit nieco ako `{% change_lang 'de' %}` kde `de` je konkretny jazyk zo `settings.LANGUAGES`, ako si ich v teste overridol.

Reply:
:ok: